apps/discovery_api/lib/discovery_api_web/auth/error_handler.ex

```diff
@@ -7,6 +7,11 @@ defmodule DiscoveryApi.Auth.ErrorHandler do
   @impl Guardian.Plug.ErrorHandler
   def auth_error(conn, error, _opts) do
     Logger.error("Auth failed: #{inspect(error)}")
-    DiscoveryApiWeb.RenderError.render_error(conn, 401, "Unauthorized")
+
+    error_message = "Unauthorized"
```

Comment:
This was actually rendering early, before the retry in verify auth0 header could hit